openIt/jsonld-contexts/openIt.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "Authorization": "https://vocab.egm.io/Authorization",
+    "Gate": "https://vocab.egm.io/Gate",
+    "Key": "https://vocab.egm.io/Key",
+    "codes": "https://vocab.egm.io/codes",
+    "from": "https://vocab.egm.io/from",
+    "initDate": "https://vocab.egm.io/initDate",
+    "name": "https://schema.org/name",
+    "rotationPeriod": "https://vocab.egm.io/rotationPeriod",
+    "status": "https://vocab.egm.io/status",
```

Comment:
`status` is defined in https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.8.jsonld, it will override yours.

Reply:
(or it may just fail now that the core context is protected)

Reply:
you can either use the term from the core context or invent a new one

Reply:
Understood. Will be creating a new term for our status, validationStatus.